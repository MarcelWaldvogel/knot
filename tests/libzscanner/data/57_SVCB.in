$ORIGIN	.
$TTL	1

; OK
@	SVCB	1	.
@	TYPE64	2	@
@	SVCB	3	x.	key1026=foo
@	TYPE64	4	.	key1026="foo bar" key1027=baz
@	SVCB	5	.	mandatory=key1,port alpn="h1,h2\,h3" port=1029
@	TYPE64	6	.	(
                                mandatory="ipv4hint"
                                alpn="h\010h"
                                ipv4hint=1.2.3.4,1.2.3.5
                                ipv6hint=1::2
                                )
@	SVCB	7	.	alpn=h1,h2\044h3 echconfig="YWhvag=="

; KO
@	SVCB	70000	.
@	SVCB	8	abcdef012345678901234567890123456789012345678901234567890123456789.
@	SVCB	9	.	key70000=foo
@	SVCB	10	.	nonextype=bar
@	SVCB	11	.	mandatory=alpn,nonextype
@	SVCB	12	.	port=70000
@	SVCB	13	.	ipv4hint=1.2.3.400
@	SVCB	14	.	ipv6hint=1::2,foo
