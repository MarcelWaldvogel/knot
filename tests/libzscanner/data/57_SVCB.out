OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=000100
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=000200
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=000301780004020003666F6F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=00040004020007666F6F206261720403000362617A
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=0005000000000400010003000100090268310568322C6833000300020405
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=0006000000000200040001000403680A680004000801020304010203050006001000010000000000000000000000000002
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0040
RDATA=000700000100090268310568322C68330005000461686F6A
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_LABEL_OVERFLOW
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_BAD_SVC_PARAM
------
WARNG=ZS_BAD_SVC_PARAM
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_BAD_IPV4
------
WARNG=ZS_BAD_ADDRESS_CHAR
------
